.github/workflows/test-go.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
       fail-fast: false
       matrix:
         # Detail: `setup-envtest list`
-        kubernetes-version: ["1.30.0", "1.31.0", "1.32.0", "1.33.0"]
+        kubernetes-version: ["1.31.0", "1.32.0", "1.33.0", "1.34.0"]
```

Comment:
Can you use same patch version as in Trainer: https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-e2e.yaml#L20